components/concourse-harbor-resource/assets/out

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+function log() {
+	echo 1>&2 "$1"
```

Comment:
Is the `1>&2` necessary given lines 12 & 13?

Reply:
Probably not ... But I'm unsure of how the streams are implemented and wanted to be sure to avoid any buffering (stdout is often line buffered whereas stderr is often not)